code/modules/mob/living/carbon/xenomorph/castes/hivelord/abilities_hivelord.dm

```diff
@@ -212,3 +212,73 @@ GLOBAL_LIST_INIT(thickenable_resin, typecacheof(list(
 	plasma_transfer_amount = PLASMA_TRANSFER_AMOUNT * 4
 	transfer_delay = 0.5 SECONDS
 	max_range = 7
+
+// ***************************************
+// *********** Xeno Jammers
+// ***************************************
+/datum/action/xeno_action/xenojammer
+	name = "Build Jammer"
+	action_icon_state = "build_jammer"
+	mechanics_text = "Create jammer, which blocks communication of infected hosts."
+	plasma_cost = 200
+	cooldown_timer = 120 SECONDS
+
+/datum/action/xeno_action/xenojammer/can_use_action(silent = FALSE, override_flags)
+	. = ..()
+	if(!.)
+		return FALSE
+	if(owner.get_active_held_item())
+		if(!silent)
+			to_chat(owner, "<span class='warning'>We need an empty claw for this!</span>")
+		return FALSE
+
+/datum/action/xeno_action/xenojammer/action_activate()
+	build_jammer(get_turf(owner))
+
+/datum/action/xeno_action/xenojammer/proc/build_jammer(turf/T)
+	var/mob/living/carbon/xenomorph/X = owner
+	var/mob/living/carbon/xenomorph/blocker = locate() in T
+	if(blocker && blocker != X && blocker.stat != DEAD)
```

Comment:
move these to can_use_action

Reply:
I believe I did this, can you check again?

Reply:
okay, I had an issue when doing this, what should I do?